tools/tracer/README.md

```diff
@@ -0,0 +1,46 @@
+Copyright(c) 2014-2020 Intel Corporation. All Rights Reserved.
+
+
+Overview
+========
+
+Media SDK Tracer is a tool which permits to dump logging information from the calls
+of the application to the Media SDK library. Trace log obtained from this tool is a
+recommended information to provide to Media SDK team on submitting questions and
+issues.
+
+
+Installation
+============
+
+Generate config file for the tracer:
+$INSTALLDIR/tools/tracer/mfx-tracer-config --default
+$INSTALLDIR/tools/tracer/mfx-tracer-config core.type file
+$INSTALLDIR/tools/tracer/mfx-tracer-config core.log ~/mfxtracer.log
+$INSTALLDIR/tools/tracer/mfx-tracer-config core.lib /opt/intel/mediasdk/lib64/libmfxhw64-p.so.1.32
```

Comment:
with LD_PRELOAD we should not have this in a config! At least on Linux.

Reply:
done